SRC/about-me/about-me.css

```diff
@@ -0,0 +1,81 @@
+.mainContainer{
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
```

Comment:
add this : `background: rgba(0, 0, 0, 0.45);`. and play with the 0.45 until you like what you see. it adds an overlay over the video so your text will pop out!